### PR TITLE
Remove deprecated Copy Task

### PR DESCRIPTION
**Pull request by karajas**
@chcosta @maririos

### CODE REVIEW

File: buildpipeline/DotNet-CoreClr-Trusted-Linux-Crossbuild.json [hunk 1]
```diff
@@ -271,30 +299,58 @@
...
       "timeoutInMinutes": 0,
+      "refName": "CopyFiles1",
       "task": {
-        "id": "1d341bb0-2106-458c-8422-d00bcea6512a",
+        "id": "5bfb729a-a7c8-4a78-a7c3-8d717bb7c13c",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "SourceFolder": "$(PB_GitDirectory)",
```
Comment (maririos) [inline buildpipeline/DotNet-CoreClr-Trusted-Linux-Crossbuild.json]: Is there a reason why we are not respecting what the previous "source folder" was and putting always `$(PB_GitDirectory)`?

File: buildpipeline/DotNet-CoreClr-Trusted-Windows-x86.json [hunk 2]
```diff
@@ -187,41 +207,71 @@
...
+      "refName": "CopyFiles1",
       "task": {
-        "id": "1d341bb0-2106-458c-8422-d00bcea6512a",
+        "id": "5bfb729a-a7c8-4a78-a7c3-8d717bb7c13c",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "SourceFolder": "$(PB_GitDirectory)",
+        "Contents": "**/*.log",
```
Comment (maririos) [inline buildpipeline/DotNet-CoreClr-Trusted-Windows-x86.json]: Same here with the `Contents` field

File: buildpipeline/DotNet-CoreClr-Trusted-Mac.json [hunk 3]
```diff
@@ -126,23 +140,49 @@
...
-        "id": "1d341bb0-2106-458c-8422-d00bcea6512a",
+        "id": "2ff763a7-ce83-4e1f-bc89-0ae63477cebe",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "CopyRoot": "",
-        "Contents": "*.log",
+        "PathtoPublish": "$(Build.StagingDirectory)\\BuildLogs",
+        "Contents": "**/*.log",
```
Comment (maririos) [inline buildpipeline/DotNet-CoreClr-Trusted-Mac.json]: hmm did you test the changes?
I don't see any `Contents` property for the task `Publish Build Artifacts`. Is it going to brake it or it just works.

**Comment by karajas** [comment]
https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build/index?buildId=1152950&_a=summary

**Pull request merged**